app/javascript/hyacinth_v1/components/batch_imports/BatchImportIndex.js

```diff
@@ -30,13 +30,19 @@ function BatchImportIndex() {
   };
 
   const onCancel = (id) => {
-    const variables = { input: { id, cancelled: true } };
-    updateBatchImport({ variables }).then(refetch);
+    // eslint-disable-next-line no-alert
+    if (window.confirm(`Are you sure you want to cancel Export Job ${id}?`)) {
+      const variables = { input: { id, cancelled: true } };
+      updateBatchImport({ variables }).then(refetch);
+    }
   };
 
   const onDelete = (id) => {
-    const variables = { input: { id } };
-    deleteBatchImport({ variables }).then(refetch);
+    // eslint-disable-next-line no-alert
+    if (window.confirm(`Are you sure you want to delete Export Job ${id}?`)) {
```

Comment:
Just realized that I actually meant to say "Batch Import" rather than "* Job", so I'm updating that now in my local copy.  Thanks for catching this though, since it definitely shouldn't say "Export".